apack.json

```diff
@@ -19,6 +19,10 @@
       "exe": "zap.app/Contents/MacOS/zap",
       "optional": true
     },
+    "zap:macosx.ARM64": {
```

Comment:
@SaharKargar
Can you rebase your changes?
There are a few changes that is not related to data-* work.

Reply:
@jingteng25742 
Rebased and pushed.